dips/dip-10.md

```diff
@@ -65,21 +76,21 @@ struct DiemIdDomain {
 }
 ```
 * Fields definition:
-   * `domain` - name of a Diem ID domain 
+   * `domain` - name of a Pay Address domain 
 * The `DiemIdDomains` resource can only be published into an account with `Role == PARENT_VASP_ROLE_ID`.
-* The `DiemIdDomains` contains a list of `DiemIdDomain` structs that are associated with a VASP. As such, it is possible to register more than one Diem ID Domain for a given VASP. We are providing this opportunity to support possible situations when two VASPs merge into one, or when a company wants to provide multiple wallet apps, while having only a single VASP parent account on the chain.
+* The `DiemIdDomains` contains a list of `DiemIdDomain` structs that are associated with a VASP. As such, it is possible to register more than one Pay Address Domain for a given VASP. We are providing this opportunity to support possible situations when two VASPs merge into one, or when a company wants to provide multiple wallet apps, while having only a single VASP parent account on the chain.
 * Only special Treasury Compliance account (address `0xB1E55ED`) can manipulate DiemIdDomains resource:
   * Only TC account can create and publish `DiemIdDomains` resource
   * Only TC account can add, remove or update an `DiemIdDomain` within `DiemIdDomains` resource
-* In order to register a Diem Id domain, a VASP needs to submit a request to Diem Association. LA will perform certain checks (out of scope of this document) before issuing an on-chain transaction to register a Diem Id Domain. These checks intend to mitigate irrelevant claims and enforce uniqueness
-* The Diem ID domain can be created as part of creating a `ParentVASP` account.
-* An entity (potentially the Association) may at some point choose to expose an open source application(*Indexer*) that will allow indexing available Diem IDs and could provide a convenient REST API for applications to fetch information about the domains. The API of such an indexer is out of scope of this RFC.
+* In order to register a Pay Address domain, a VASP needs to submit a request to Diem Association. LA will perform certain checks (out of scope of this document) before issuing an on-chain transaction to register a Pay Address Domain. These checks intend to mitigate irrelevant claims and enforce uniqueness
+* The Pay Address domain can be created as part of creating a `ParentVASP` account.
+* An entity (potentially the Association) may at some point choose to expose an open source application(*Indexer*) that will allow indexing available Pay Addresss and could provide a convenient REST API for applications to fetch information about the domains. The API of such an indexer is out of scope of this RFC.
 
-## Diem ID domain events
+## Pay Address Domain Events
```

Comment:
here's where we state that an application can be built to monitor for these events to construct a mapping of pay address domain to vasp account